core/edit_team.go

```diff
@@ -0,0 +1,68 @@
+package core
+
+import (
+	"github.com/nlopes/slack"
+	log "github.com/sirupsen/logrus"
+	"github.com/ubclaunchpad/rocket/cmd"
+	"github.com/ubclaunchpad/rocket/model"
+)
+
+// NewEditTeamCmd returns an add team command that creates a new Launch Pad team
+func NewEditTeamCmd(ch cmd.CommandHandler) *cmd.Command {
+	return &cmd.Command{
+		Name:     "edit-team",
+		HelpText: "Update an existing Launch Pad team",
+		Options: map[string]*cmd.Option{
```

Comment:
can we set a team description too?

Reply:
Yea we could add that, but in a separate PR.